kibbeh/src/ui/SettingsItemButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { BaseSettingsItem } from "./BaseSettingsItem";
+import { Button } from "./Button";
+
+export type SettingsItemButtonProps = {
+  text: string,
+  buttonText: string,
+  disabled?: boolean,
+  onClick: (e: React.MouseEvent | React.TouchEvent) => void,
+};
+
+export const SettingsItemButton: React.FC<SettingsItemButtonProps> = ({
+  text,
+  buttonText,
+  disabled = false,
+  onClick,
+}) => {
+  return (
+    <BaseSettingsItem className="p-4">
+      <p className="text-primary-200 mb-4">{text}</p>
+      <Button color="secondary-accent" size="small" onClick={onClick} disabled={disabled} className="py-2" >
+        <p className="font-bold">
+          {buttonText}
+        </p>
+      </Button>
+    </BaseSettingsItem>
+
```

Comment:
Extra blank line

Reply:
Thank you, resolved in 
https://github.com/benawad/dogehouse/pull/2272/commits/524a88d1881d89c95920a4c92f92df36585fca94